paymentsheet/src/main/java/com/stripe/android/link/confirmation/DefaultLinkConfirmationHandler.kt

```diff
@@ -115,10 +115,10 @@ internal class DefaultLinkConfirmationHandler @Inject constructor(
         linkAccount: LinkAccount,
         cvc: String?
     ): ConfirmationHandler.Args {
-        val confirmationOption = if (paymentDetails.useLinkCardBrandConfirmation) {
-            LinkCardBrandConfirmationOption(
+        val confirmationOption = if (configuration.passthroughModeEnabled) {
```

Comment:
This looks correct to me: We want `type: link` for payment method mode, and `type: card` for passthrough mode 👍